jobs/transforms/blog_validation.py

```diff
@@ -47,8 +47,8 @@ class ValidateBlogPostModelDomainObjectsInstances(
         base_validation.ValidateModelDomainObjectInstances):
     """Provides the validation type for validating blog post objects."""
 
-    def _get_model_domain_object_instance(self, blog_post_model):
-        # type: (Any) -> blog_domain.BlogPost
+    def _get_model_domain_object_instance(
+            self, blog_post_model: Any) -> blog_domain.BlogPost:
```

Comment:
I think we're using too much "Any" in this file. Might be a good idea to scan this PR for use of Any and make sure that each case has a proper justification.

Any thoughts on how to avoid this in the future, too? It is the "easy way out", and people will use it if you don't prevent it.

Reply:
@seanlip I have scanned all the files in this PR, and made sure all the Any's have explanation.

To avoid this in future, I think a lint check one possible way, but I am not sure how easy that would be to implement or how effective that can be - for that we need to define exactly where wrt to the Any its explanatory comment would be found. For  type hints written for a variable it can directly above the variable, but for function defs spanning multiple lines it's difficult to verify whether the comment just above it is the explanation or something else - I agree most functions won't have comments above them and the check might cover atleast some genuine cases, but it's hard to verify whether all the Any's (in case of multiple ones in a func def) have been explained or not. If we define a pattern for explaining Any's it might be possible. I am thinking of creating an issue for it now instead of dealing with it in this PR. Codeowners can be advised to take care of it till then perhaps?